Tasks/JavaToolInstallerV0/javatoolinstaller.ts

```diff
@@ -1,12 +1,16 @@
 import fs = require('fs');
+import os = require('os');
 import path = require('path');
 import taskLib = require('azure-pipelines-task-lib/task');
 import toolLib = require('azure-pipelines-tool-lib/tool');
 
-import { AzureStorageArtifactDownloader } from "./AzureStorageArtifacts/AzureStorageArtifactDownloader";
-import { JavaFilesExtractor } from './FileExtractor/JavaFilesExtractor';
-import {BIN_FOLDER} from "./FileExtractor/JavaFilesExtractor";
+import { AzureStorageArtifactDownloader } from './AzureStorageArtifacts/AzureStorageArtifactDownloader';
+import { JavaFilesExtractor, BIN_FOLDER } from './FileExtractor/JavaFilesExtractor';
+import { sleepFor, sudo, attach, detach } from './taskutils';
```

Comment:
I think that it may be replaced by:
`import taskutils = require('./taskutils');`
and then method calls would look like this:
`taskutils.sleepFor()`

Reply:
Changed, thanks!